### DISCUSSION TITLE
Does it make sense to add `swaggertype` into tag automatically?

### DISCUSSION THREAD

**Discussion by abawchen** [aarondl/sqlboiler#1155] [Ideas]
The idea comes from https://github.com/swaggo/swag/issues/985, that I want to auto-generate a more clear swagger document when using `null`.

Following is an example and its generated document:
```go
type foo struct {
    IsEmpty null.Bool
}
```
```json
{
    "IsEmpty": {
          "bool": true,
          "valid": true
    }
}
```

I think it would be good if sqlboiler can add `swaggertype` automatically according to its type, say:
```go
type foo struct {
    IsEmpty null.Bool `..... swaggertype:"bool"`
}
```
```json
{
    "IsEmpty": bool
}
```

**Comment by stephenafamo** [comment]
If there is a PR with a clean way to optionally enable this, I'll be happy to review and merge.
